docs/site/BelongsTo-relation.md

```diff
@@ -179,3 +179,42 @@ with the name following the pattern `__{methodName}__{relationName}__` (e.g.
 `Order.__get__customer`). While we recommend to create a new controller for each
 hasMany relation in LoopBack 4, we also think it's best to use the main CRUD
 controller as the place where to explose `belongsTo` API.
+
+## Notes
+### Recursive relation
```

Comment:
Let's merge these two headings into a single one please.

```markdown
## Handling recursive relations
```

Reply:
Updated. Thank you.